script/release-ts.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+API_URL="https://api.github.com"
+
```

Comment:
add check if `$GITHUB_TOKEN"` is set and error if not

Reply:
same for `$GITHUB_ACTION_REPOSITORY` and `"$AKASH_TS_ROOT"`

Reply:
done